starter-code/public/new.html

```diff
@@ -85,9 +85,10 @@
     <script src="vendor/scripts/highlight.pack.js"></script>
     <script src="scripts/article.js"></script>
     <script src="scripts/articleView.js"></script>
-    <!-- TODO: Due to changes in article.js and articleView.js to attach their interior objects to a global 'app' object, we need to make changes to the contents of the following <script> tag: -->
+    <!-- ????: Due to changes in article.js and articleView.js to attach their interior objects to a global 'app' object, we need to make changes to the contents of the following <script> tag: -->
     <script>
       articleView.initNewArticlePage();
```

Comment:
Just like you did with the index page, don't we also now need to grab articleView from our global app object?